Sources/xcodeproj/Objects/Project/PBXObject.swift

```diff
@@ -71,12 +71,10 @@ public class PBXObject: Hashable, Decodable, Equatable, AutoEquatable {
     /// Returns the objects the object belong to.
     ///
     /// - Returns: objects the object belongs to.
-    /// - Throws: an error if this method is accessed before the object has been added to a project.
-    func objects() throws -> PBXObjects {
-        guard let objects = self.reference.objects else {
-            let objectType = String(describing: type(of: self))
-            throw PBXObjectError.orphaned(type: objectType, reference: reference.value)
-        }
-        return objects
+    func objects() -> PBXObjects {
```

Comment:
We should not allow orphaned objects that don't belong to any project. We are not enforcing that from the API but such enforcement will land soon.